ci/manylinux2014_x86_64-vcpkg-gdal.Dockerfile

```diff
@@ -13,6 +13,13 @@ RUN bootstrap-vcpkg.sh && \
 
 COPY ci/custom-triplets/x64-linux-dynamic.cmake opt/vcpkg/custom-triplets/x64-linux-dynamic.cmake
 COPY ci/vcpkg-custom-ports/ opt/vcpkg/custom-ports/
+COPY ci/vcpkg.json opt/vcpkg/
 
 ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/vcpkg/installed/x64-linux-dynamic/lib"
-RUN vcpkg install gdal[core]:x64-linux-dynamic --overlay-triplets=opt/vcpkg/custom-triplets --overlay-ports=opt/vcpkg/custom-ports
+RUN vcpkg install --overlay-triplets=opt/vcpkg/custom-triplets \
+    --triplet=x64-linux-dynamic \
+    --overlay-ports=opt/vcpkg/custom-ports \
+    --feature-flags="versions,manifests" \
+    --x-manifest-root=opt/vcpkg \
+    --x-install-root=opt/vcpkg/installed && \
```

Comment:
Question: the reason that it is now needed to mention the install-root, is that because using manifest mode changes that? (it would now try to install inside the project, instead of in the vcpkg root?)

Reply:
Yes, using manifest mode changes this, and would have put it in `ci/vcpkg_installed/` which then would have required other updates here to use that path.